source/h1_encoder.c

```diff
@@ -403,10 +403,7 @@ int aws_h1_encoder_message_init_from_response(
         goto error;
     }
 
-    err = aws_byte_buf_init(&message->outgoing_head_buf, allocator, head_total_len);
-    if (err) {
-        return AWS_OP_ERR;
-    }
+    aws_byte_buf_init(&message->outgoing_head_buf, allocator, head_total_len);
```

Comment:
It looks like aws_byte_buf_init could return an AWS_OP_ERR resulting in a zeroed out outgoing_head_buf. Just making sure we shouldn't be checking the return and goto error here to clean up message if that happens.

Reply:
In the past, aws_buf_buf_init() could fail if the allocator failed. But a few years back, we made it impossible for memory allocation to fail (the program will abort before the allocator can return NULL)